wgpu-core/src/validation.rs

```diff
@@ -680,7 +681,7 @@ impl Interface {
         list.push(varying);
     }
 
-    pub fn new(module: &naga::Module, analysis: &naga::proc::analyzer::Analysis) -> Self {
+    pub fn new(module: &naga::Module, info: &naga::valid::ModuleInfo) -> Self {
         let layouter = Layouter::new(&module.types, &module.constants);
```

Comment:
one of the things we need to change now is using `info.layouter` instead of creating a new one